schema/artist/index.js

```diff
@@ -65,6 +65,9 @@ const ShowField = {
     is_reference: {
       type: GraphQLBoolean,
     },
+    visible_to_public: {
+      type: GraphQLBoolean,
+    },
```

Comment:
So these are all the different arguments that resolving the `shows` field on an artist can take. Since shows for an artist are currently powered by `/api/v1/related/shows`, these correspond to the possible API params for that endpoint.

These are actually referenced/used [here](https://github.com/artsy/metaphysics/blob/3ba358eadcaf6fc6cc810e8cc7628e89656c3f88/schema/artist/index.js#L482-L484).